src/elements/tables/table.md

```diff
@@ -0,0 +1,29 @@
+```jsx
+import { TableHead, TableRow, TableHeadCell, TableCell } from '.';
+
+<Table>
+  <TableHead>
```

Comment:
It would be nice to use a new markup, i.e. `<Table.Row />`

Reply:
It's not a big deal now. You know that you can use it in such way. For this moment this is enough 😉